core/src/main/java/org/kiwix/kiwixmobile/core/bookmark/BookmarksActivity.java

```diff
@@ -138,6 +143,15 @@ protected void onCreate(Bundle savedInstanceState) {
     setupBookmarksAdapter();
     recyclerView.setAdapter(bookmarksAdapter);
 
+    bookmarksSwitch.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
+      @Override public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
+        sharedPreferenceUtil.setShowHistoryCurrentBook(!isChecked);
+        presenter.loadBookmarks(sharedPreferenceUtil.getShowHistoryCurrentBook());
```

Comment:
these are the history methods, not the bookmarks

Reply:
Apologies. I don't know how I missed this.

Reply:
An easy copy paste mistake to make